packages/components/package.json

```
@@ -45,6 +45,10 @@
 		"@wordpress/primitives": "file:../primitives",
 		"@wordpress/rich-text": "file:../rich-text",
 		"@wordpress/warning": "file:../warning",
+		"@wp-g2/components": "^0.0.113",
+		"@wp-g2/styles": "^0.0.113",
+		"@wp-g2/substate": "^0.0.113",
+		"@wp-g2/utils": "^0.0.113",
```

We need only the following so far:
```suggestion
		"@wp-g2/components": "^0.0.113",
```